src/lib/assetBridger/erc20Bridger.ts

```diff
@@ -369,9 +375,19 @@ export class Erc20Bridger extends AssetBridger<
 
     const l2Address = await l2GatewayRouter.calculateL2TokenAddress(l1Address)
     if (l2Address.toLowerCase() !== erc20L2Address.toLowerCase()) {
-      throw new ArbSdkError(
-        `Unexpected l1 address. L1 address from token is not registered to the provided l2 address. ${l1Address} ${l2Address} ${erc20L2Address}`
+      // also check with the standard gateway
```

Comment:
Why also check with the standard gateway? Is this because we want to not throw if the l2address is the standard token, but a custom token has already been registered over the top?

Reply:
That is what I am thinking, would make bridging back those token to L1 easier. Or do you think we should simply throw?

Reply:
Yeh, I think what you have makes sense. We're trying to protect against fraud here I guess, and that's not possible with the standard token. 

I think just add more comments, it took me a bit of thinking to figure out why we wanted to do this

Reply:
On second thought I think we might just ignore this and throw. Even if we let it pass validation here it still need to use the StandardGateway manually, instead of the router.